reapy/core/project/project.py

```diff
@@ -7,9 +7,7 @@
 
 
 class Project(ReapyObject):
-
     """REAPER project."""
-

```

Comment:
I don't think there is any PEP recommendation indicating these blank lines should be removed, is there? I kind of prefer the way it was before, as it is consistent with the blank lines between methods definitions and avoids a heavy pack at the top of the class definition. Similarly to my previous comment, I'm not sure it is relevant to start changing these in the whole project, unless you have a good reason for it.